bot/config.py

```diff
@@ -0,0 +1,15 @@
+import dataclasses
+import typing
```

Comment:
For consistency across the project, use `import typing as t`

Reply:
Done